python/lsst/ts/aos/utils/m1m3/m1m3_aav.py

```diff
@@ -361,6 +370,61 @@ def vect_to_fam(row: pd.Series) -> pd.Series:
         )
         self.calculated_fam.set_index(self.fitter.aav.index, inplace=True)
 
+    async def load_accelerometers(self, start: Time, end: Time) -> None | pd.DataFrame:
+        """Load DC accelerometers data in given interval. Those are used for
+        acceleration  corrections.
+
+        Parameters
+        ----------
+        start : Time
+            Search interval start time.
+        end : Time
+            Search interval end time.
+
+        Returns
+        -------
+        ret : pd.DataFrame
+            Time indexed dataframe with measured DC accelerometers values
+            (angularAcceleration[XYZ]).
+        """
+        logging.debug(f"Retrieving accelerometer data for {start.isot} - {end.isot}..")
+        ret = await self.client.select_time_series(
```

Comment:
I suggest you change the name to something that actually refers to what this object holds

Reply:
Changed to accelerometers